homeassistant/components/media_player/intent.py

```diff
@@ -0,0 +1,48 @@
+"""Intents for the media_player integration."""
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    SERVICE_MEDIA_NEXT_TRACK,
+    SERVICE_MEDIA_PAUSE,
+    SERVICE_MEDIA_PLAY,
+    SERVICE_VOLUME_SET,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import intent
+import homeassistant.helpers.config_validation as cv
+
+from . import ATTR_MEDIA_VOLUME_LEVEL, DOMAIN
+
+INTENT_MEDIA_PAUSE = "HassMediaPause"
+INTENT_MEDIA_UNPAUSE = "HassMediaUnpause"
+INTENT_MEDIA_NEXT = "HassMediaNext"
+INTENT_SET_VOLUME = "HassSetVolume"
```

Comment:
This service made me wonder … how will we respond if the specified entity does not support the set volume service?

Reply:
Hmmm...we may want to consider an "unsupported feature" error message.

Reply:
We already raise that error from service helper

Reply:
Just would need to translate it for voice